### DISCUSSION TITLE
How to get products names translated into different languages ?

### DISCUSSION THREAD

**Discussion by Tbz8** [Intechnity-com/OdooJsonRpcClient#33] [Q&A] [answered]
Hello,

I need to retrieve the names of the products. However, the product names are translated into several languages.Unfortunately I can't get the values even when specifying the "language" (en_EN / fr_FR) in the OdooConfig object. 

Is there a trick to perform this kind of request with this NuGet ? 

Thanks for your help !

**Comment by patricoos** [comment, marked as answer]
At first add language to odoo that U are interested in:
Settings => General Settings => Languages => Add Language

This test works for me (Dutch / Nederlands) => [Get_name_in_nl](https://github.com/patricoos/PortaCapena.OdooJsonRpcClient/blob/335fcee9b987779c5b456886e5d78eb910283f41/PortaCapena.OdooJsonRpcClient.Example/ProductTemplateOdooModelRepositoryTests.cs#L61)

**Reply by Tbz8** [reply]
Hi, 

Thanks for you reply ! It works well :-)
